src/FileCollection/FileCollection.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useState, useEffect } from "react";
+import { useDropzone } from "react-dropzone";
+
+import styled from "styled-components";
+import { useContent, WithContentProps } from "../Content/Content";
+import { AiOutlineClose } from "react-icons/ai";
+import fileDropIcon from "./file-drop.png";
+import { colors } from "../shared/constants";
+import { SpawnSyncOptionsWithStringEncoding } from "child_process";
+
+const StyledOuterContainer = styled.div``;
+
+const StyledContainer = styled.div`
+  height: ${props => (props.filesUploaded ? "40px" : "110px")};
+  display: flex;
+  align-items: center;
+  border: ${props =>
+    props.isDragActive
+      ? `1px solid ${props.theme.colors.primary.light || colors.LIGHT_BLUE}`
+      : `1px dashed ${props.theme.colors.primary.medium || colors.GRAY}`};
+  border-radius: 20px;
+  margin: 10px 0px;
+  justify-content: ${props => (props.isDragActive ? "center" : "auto")};
+  background-color: ${props =>
+    props.isDragActive
+      ? `${props.theme.colors.primary.light || colors.LIGHT_BLUE}`
+      : "#ffffff"};
+`;
+
+const StyledDropzoneText = styled.p`
+  width: 100%;
+  display: flex;
+  flex-direction: ${props => (props.filesUploaded ? "row" : "column")};
+  align-items: center;
+  justify-content: ${props => (props.filesUploaded ? "space-between" : "auto")};
+  line-height: 25px;
+  margin: 10px 20px;
+`;
+
+const StyledDropHereText = styled.span`
+  display: flex;
+  flex-direction: ${props => (props.filesUploaded ? "row" : "column")};
+  align-items: center;
+  color: ${props =>
+    props.isDragActive
+      ? props.theme.colors.primary.dark || colors.BLUE_DENIM
+      : "#65666d"};
+`;
+
+const StyledBrowseFileText = styled.span`
+  color: ${props => props.theme.colors.secondary.medium || colors.BLUE_DENIM};
+`;
+
+const StyledFileContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 15px;
+  height: 30px;
+  border: 1px solid #d9d9e3;
+  border-radius: 35px;
+  margin-bottom: 15px;
+`;
+
+const StyledFileNames = styled.a`
+  color: #65666d;
+  text-decoration: none !important;
+`;
+
+const StyledRemoveButton = styled.button`
+  width: 5%;
+  border: none;
+  font-size: 20px;
+  background-color: transparent;
+  color: ${props => props.theme.colors.primary.dark};
+`;
+
+const StyledDropIcon = styled.img`
+  height: 20px;
+  width: 16px;
+  margin: 5px 20px;
+`;
+
+const StyledError = styled.div`
+  color: red;
+  margin: 0px;
+  padding-left: 17px;
+`;
+
+export interface FileCollectionProps extends WithContentProps {
+  acceptedFiles: string;
+  rejectFileError: string;
+  maxSize: number;
```

Comment:
1. Let's add all the other props (e.g., onChange, onDropVerbiage, ...) here
2. While doing so, let's think about props that are optional and those that will have default values.

E.g., It could be a good idea to have a default alue for `maxSize`. Our backend has 15mb as the default max size for some stuff.

```javascript
maxSize?: number;

...

maxSize = 15000000,
...rest
```